SnowballSwiftExample/SnowballSwiftExample/Example Views/SnowballSwiftExampleTextInputView.swift

```diff
@@ -14,19 +14,19 @@ struct SnowballSwiftExampleTextInputView: View {
 
     var body: some View {
         VStack(alignment: .leading) {
-            SnowballTextInput($textInput, title: "Label", subtitle: "Provide additional context or helpful information.")
+            TextInput($textInput, title: "Label", subtitle: "Provide additional context or helpful information.")
 
-            SnowballTextInput($textInput, title: "Label")
+            TextInput($textInput, title: "Label")
 
-            SnowballTextInput($textInput)
+            TextInput($textInput)
 
-            SnowballTextInput($nonEmptyTextInput)
+            TextInput($nonEmptyTextInput)
 
-            SnowballTextInput($textInput, rightSystemIcon: "clipboard")
+            TextInput($textInput, rightSystemIcon: "clipboard")
 
-            SnowballTextInput($textInput, leftSystemIcon: "clipboard")
+            TextInput($textInput, leftSystemIcon: "clipboard")
 
-            SnowballTextInput($textInput, placeholder: "placeholder")
+            TextInput($textInput, placeholder: "placeholder")
```

Comment:
this works but ehhhhh, it breaks a lot of rules around styling. let me talk with a friend and I will find out more about how they do their custom styling and perhaps if we can borrow for TextField.

Reply:
deleting the text input fr now